src/main/resources/application.conf

```diff
@@ -185,6 +186,7 @@ scorex {
     maxInvObjects = 400
     desiredInvObjects = 400
     nodeName = "ergo-testnet-2.0"
+    nodeName = ${?NODENAME}
```

Comment:
Not sure if previous `nodeName = "ergo-testnet-2.0"` overwrited to empty string if NODENAME not set